sdk/resourcehealth/azure-resourcemanager-resourcehealth/pom.xml

```diff
@@ -95,5 +95,11 @@
       <version>2.45.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-compute;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-resources</artifactId>
```

Comment:
same, compute contains it

Reply:
Fixed by [commit#7d90a3c](https://github.com/Azure/azure-sdk-for-java/pull/43440/commits/7d90a3ca278546258c5ce281dd8ffba04826f047)